### ISSUE TITLE
img missing alt text error is not caught by Section508 guidelines

### ISSUE THREAD

**Issue by shanear** [squizlabs/HTML_CodeSniffer#153]
<img width="1040" alt="screenshot 2016-02-07 16 20 13" src="https://cloud.githubusercontent.com/assets/1596259/12875392/476b8e70-cdb7-11e5-8431-75687a1621aa.png">

**Comment by lwright-sq** [comment]
Hi Shane,

Thanks for the heads up. I can see where the problem is. I've fixed this in the trunk and it shouldn't be too long before it's released into the auditor. Thanks again!

**Comment by shanear** [comment]
Awesome! thanks for fixing Luke.
